compiler/rustc_borrowck/src/region_infer/mod.rs

```diff
@@ -327,11 +327,13 @@ fn sccs_info<'tcx>(infcx: &BorrowckInferCtxt<'tcx>, sccs: &ConstraintSccs) {
     let mut var_to_origin_sorted = var_to_origin.clone().into_iter().collect::<Vec<_>>();
```

Comment:
btw, the sorted vec is also only used for the debug! call.

Note that `scc_infos` is not called in prod, it's under a `cfg!(debug_assertions)`. I guess you could switch that to check the log level instead (or inline it there) to make it clearer.

Reply:
ugh, so i was only hitting this bc of my own debug assertions being enabled; i should've looked lol